### ISSUE TITLE
Copy function not working

### ISSUE THREAD

**Issue by tarukofusuki** [philc/vimium#4197]
Copy function not working. 

Not working both with "YY" and "YF" shortcuts

Brave 1.46.144 Chromium: 108.0.5359.128 (64 bit)  and Windows 10 Pro version 19044.2364, Vimium version 1.67.4

**Comment by jakobkogler** [comment]
Same problem here (since a couple of days):

yy shows a popup with "Yanked https://github.com/...", but it actually doesn't copy the url. The previously copied value is still copied.

Firefox 108.0.1 (64-bit) 
5.15.81-1-MANJARO
vimium 1.67.4

**Comment by solzard** [comment]
Exactly the same: `yy` shows a popup, but nothing is copied.

Brave Version 1.46.144 Chromium: 108.0.5359.128 (Official Build) (arm64)
Vimium 1.67.4
macOS 12.6.2

The console got an error below
```console
hud.js:152 Uncaught (in promise) DOMException: Must be handling a user gesture to use custom clipboard
(anonymous) @ hud.js:152
setTimeout (async)
setTimeout @ utils.js:278
copyToClipboard @ hud.js:145
(anonymous) @ hud.js:174
ownerPagePort.onmessage @ ui_component_server.js:23

```

**Comment by palmerusaf** [comment]
It looks like they rolled back to version 1.64.6 on the firefox add-ons store. Try removing the extension by right clicking on it and clicking remove, then re-downloading it. This worked for me.

**Comment by philc** [comment]
The Firefox issue should be fixed in 1.67.5, which is now on the Firefox store. Brave support is being tracked in #4200, so I'll close this.
